api-tester/build.gradle

```diff
@@ -17,7 +17,7 @@ android {
     }
 
     defaultConfig {
-        minSdkVersion 21 // Compose requires minSdkVersion 21
+        minSdkVersion 24 // RevenueCat UI requires 24
```

Comment:
Had to update the api tests in order to test the revenuecatui module. I think it should be fine, since our API shouldn't depend on the sdk version

Reply:
Hmm but this helped guarantee that we didn't break this compatibility. Should we create a separate module for `revenuecatui` API tests?

Reply:
Hmm I see your point, but it's difficult to change the minSdkVersion accidentally 🤞, and the main sdk shouldn't compile already if there is any issue that would be caught by the API tester. So I think it's ok to keep it in the same module? cc @vegaro. In case you have other thoughts.